profile/Client/profileClient.go

```diff
@@ -379,17 +380,37 @@ func AggregateSummary(inputMap map[Profile]*Frequency, Operation string) map[Pro
 
 func convertToJSON(Operation string, data []Profile) {
 	var jsonArray []string
-	jsonByte, _ := json.MarshalIndent(data, " ", "   ")
+	jsonByte, err := json.MarshalIndent(data, " ", "   ")
+	if err != nil {
+		log.Fatal("Cannot marshal JSON", err)
+	}
+
 	//unmarshalling here because it is marshalled two times for some reason
 	if err := json.Unmarshal(jsonByte, &jsonArray); err != nil {
-		fmt.Println("Error parsing JSON array:", err)
+		log.Fatal("Cannot unmarshal JSON", err)
 	}
+
 	if len(jsonArray) > 0 {
-		filepath := "Profile_Summary/"
-		err := os.MkdirAll(filepath, 0600)
-		err = os.WriteFile(filepath+Operation+".json", []byte(jsonArray[0]), 0600)
+		directory := "Profile_Summary/"
```

Comment:
```suggestion
		directory := "profile_summary/"
```

Can we make this value configurable like how we handle it in `karmor recommend`? Ref 
https://github.com/kubearmor/kubearmor-client/blob/9ad1b9993b85b3e58b151aefacf8ac5ac34108c5/cmd/recommend.go#L49